src/components/tile/tile.stories.mdx

```diff
@@ -257,6 +256,52 @@ Please note that it will not have any effect if the Tile orientation is set to `
   </Story>
 </Canvas>
 
+### With custom heights
+
+The Tile component can also have a custom `height` which accepts either numbers or strings: any decimal between 0 and 1.0
+will be converted into a percentage, any number grater will be be treated as pixels and any valid CSS string will also work.
```

Comment:
```suggestion
will be converted into a percentage, any number greater will be be treated as pixels and any valid CSS string will also work.
```

Reply:
It seems we had a similar typo for the `With custom widths` description too, would you be happy to amend this @edleeks87?

Reply:
Yep that's what I get for copying and pasting 🤦